meta/data.go

```diff
@@ -662,6 +662,26 @@ func (di DatabaseInfo) RetentionPolicy(name string) *RetentionPolicyInfo {
 	return nil
 }
 
+// ShardInfos returns a list of all shards' info for the database.
+func (di DatabaseInfo) ShardInfos() []ShardInfo {
```

Comment:
Both are doing the same thing.  If his merges first, then I'll update his to call this func or if this merges first, he can do the update.